Registry/AssetProcessorPlatformConfig.setreg

```diff
@@ -87,24 +87,60 @@
+                //
                 // If specified, output will be prepended to every path found in that recognizer's watch folder.
+                //
                 // Note that you can also make the scan folder platform specific by using the keywords include and exclude.
+                //
                 // Both include and exclude can contain either platform tags, platform identifiers or both.
                 // if no include is specified, all currently enabled platforms are included by default.
                 // If includes ARE specified, it will be filtered down by the list of currently enabled platforms. 
                 // "ScanFolder (unique identifier)": {
-                //  "include": "(comma seperated platform tags or identifiers)",
-                //  "exclude": "(comma seperated platform tags or identifiers)"
+                //    "watch": "(the relative path to scan for source assets)",
+                //    "display": "(how to display this scan folder in UX)",
+                //    "order": (some number where lower is more important),
+                //    "recursive": (0 to only scan this folder, 1 to scan other children folders),
+                //    "include": "(comma separated platform tags or identifiers)",
+                //    "exclude": "(comma separated platform tags or identifiers)"
                 // }
                 // For example if you want to include a scan folder only for platforms that have the platform tags tools and renderer 
-                // but omit it for platform mac, you will have a scanfolder rule like
+                // but omit it for platform mac, you will have a scan folder rule like
                 // "ScanFolder (unique identifier)": {
-                // "watch": "@ROOT@/foo",
-                // "include": "tools, renderer",
-                // "exclude": "mac"
+                //   "watch": "@ROOT@/foo",
+                //   "include": "tools, renderer",
+                //   "exclude": "mac"
                 // }
+                //
+                // If the source assets need to be separated by platform, then the root project folder should place the assets in 
+                // separate folders then exclude the other platforms in those scan folders. For example if the target asset platforms 
+                // are Windows-PC, Android, and iOS this could be the layout of the scan folders:
+                //"ScanFolder Project/Assets/Shared": {
+                //    "watch": "@PROJECTROOT@/Shared",
+                //    "display": "@PROJECTNAME@/Shared",
+                //    "recursive": 1
+                //},
+                //"ScanFolder Project/Assets/Windows": {
+                //    "watch": "@PROJECTROOT@/Windows",
+                //    "display": "@PROJECTNAME@/Windows",
+                //    "recursive": 1,
+                //    "exclude": "ios,android"
```

Comment:
I thought the original issue was that someone had put a folder under @PROJECTROOT@ and added excludes to it when the parent (@PROJECTROOT@) was already including other things.  Or did I misunderstand why that wasn't working?